deploy/operator_ironic.yaml

```diff
@@ -102,7 +116,7 @@ spec:
                   name: mariadb-password
                   key: password
         - name: ironic-httpd
-          image: quay.io/metalkube/metalkube-ironic
+          image: quay.io/imain/ironic:interface
```

Comment:
Presume this is not intentional?

Reply:
correct, I was doing my own builds at one point